packages/react/src/analytics/integrations/Vitorino/providers/Riskified/types/types.ts

```diff
@@ -0,0 +1,8 @@
+import type { VitorinoIntegrationBaseOptions } from '../../../types';
+
+export interface RiskifiedOptions {
+  siteId?: string;
```

Comment:
I noticed both `RiskifiedOptions` and `ForterOptions` have the `siteId` property. If they are supposed to have the same value, shouldn't this option be in `VitorinoIntegrationBaseOptions`?

Reply:
Never mind, just realized that they are supposed to be different.